tests/networking/tests/networking_multus_links_negative.go

```diff
@@ -0,0 +1,127 @@
+package tests
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/networking/nethelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/networking/netparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Networking custom namespace, ", func() {
```

Comment:
also here 
did you mean 
Describe("Networking custom namespace, ", func() - > Describe("Networking custom namespace", func() ?
just deleted the comma

Reply:
Check my comment above